src/module_wrap.cc

```diff
@@ -517,7 +517,7 @@ static MaybeLocal<Promise> ImportModuleDynamically(Local<Context> context,
       Local<Value> error = v8::Exception::Error(
         OneByteString(iso, "invalid_context"));
       if (resolver->Reject(context, error).IsJust()) {
-        return resolver.As<Promise>();
+        return handle_scope.Escape(resolver.As<Promise>());
```

Comment:
@TimothyGu I admit that I'm not 100% familiar with proper `HandleScope` handling - but I assumed that I had to go back to an `EscapableHandleScope` because I'm now creating a new object (`Promise::Resolver::New` above) in this code path? Happy to revert if this isn't necessary. :)

Reply:
@jkrems Yep I believe this is necessary now. Thanks for checking!